backend/app/api/endpoints/adapter/subscription_follows.py

```
@@ -0,0 +1,343 @@
+        db=db,
+        subscription_id=subscription_id,
+        user_id=current_user.id,
+        skip=skip,
+        limit=limit,
+    )
+
+
+@router.get("/{subscription_id}/followers/count")
+def get_followers_count(
+    subscription_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)